src/components/DialogContentContainer/__stories__/DialogContentContainer.stories.mdx

```diff
@@ -0,0 +1,57 @@
+import DialogContentContainer from "../DialogContentContainer";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import { createComponentTemplate } from "../../../storybook/functions/create-component-story";
+import DialogContentContainerExample from "./DialogContentContainerExample";
+import "./DialogContentContainer.stories.scss";
+
+<Meta title="Popover/DialogContentContainer" component={DialogContentContainer} />
+
+# DialogContentContainer
+
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Variants](#variants)
+- [Feedback](#feedback)
+
+## Overview
+
+Please write here the component description
+
+<Canvas>
+  <Story name="Overview">
+    <DialogContentContainer>
+      <DialogContentContainerExample />
+    </DialogContentContainer>
+  </Story>
+</Canvas>
+
+## Props
+
+<ArgsTable of={DialogContentContainer} />
+
+## Usage
+
+<UsageGuidelines guidelines={["Use this component in order to wrap components within Dialog or Modal"]} />
+
+## Variants
+
+### Popover
+
+<Canvas>
+  <Story name="Popover">
+    <DialogContentContainer type={DialogContentContainer.types.POPOVER}>
+      <DialogContentContainerExample />
```

Comment:
same comment about the templates (for more readable code inside the "show code" section)